lib/sync/index.js

```diff
@@ -89,38 +106,73 @@ function setLogLevel(dataset_id, params, cb) {
   }
 }
 
-function sync(dataset_id, params, callback) {
+function sync(dataset_id, params, cb) {
   syncUtil.doLog(dataset_id, 'info', 'sync');
 
   // TODO
 
   return cb(null, {});
 }
 
-/* Synchronise the individual records for a dataset */
-function syncRecords(dataset_id, params, callback) {
+// Synchronise the individual records for a dataset
+function syncRecords(dataset_id, params, cb) {
   syncUtil.doLog(dataset_id, 'info', 'syncRecords');
 
   // TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1135)